pkg/securitypolicy/framework.rego

```diff
@@ -70,7 +70,7 @@ overlay_mounted(target) {
 default candidate_containers := []
 
 candidate_containers := containers {
-    semver.compare(data.policy.framework_svn, svn) == 0    
+    semver.compare(data.policy.framework_svn, svn) == 0
```

Comment:
unrelated, but is there a rego linter/formatter we can add to the pipeline?

Reply:
not that i am aware of

Reply:
https://github.com/open-policy-agent/opa/issues/3218

Reply:
and someone has a project that allows you to write rego that are your linting rules:

https://github.com/m-mizutani/regolint

Reply:
ohh, I was curious to see if we can avoid formatting changes showing up in the diffs, but we can revisit in the future